deploymentTemplates/aksarc-ARM/HCI/azuredeploy.json

```diff
@@ -133,9 +126,81 @@
             "metadata": {
                   "description": "Fully qualified custom location ARM resource Id."
               }
+          },
+          "lnetName": {
+            "type": "string",
+            "metadata": {
+                  "description": "Name of the Logical Network."
+              }
+          },
+          "dnsServers": {
+            "type": "string"
+          },
+          "addressPrefix": {
+            "type": "string"
+          },
+          "ipAllocationMethod": {
+            "type": "string"
+          },
+          "vippoolstart": {
+            "type": "string"
+          },
+          "vippoolend": {
+            "type": "string"
+          },
+          "nextHopIpAddress": {
+            "type": "string"
           }
       },
       "resources": [
+        {
+        "type": "Microsoft.AzureStackHCI/logicalNetworks",
+        "apiVersion": "2025-02-01-preview",
```

Comment:
Is there a way to parameterize the API version from ARM template itself? Otherwise, prefer the latest GA and non-preview API versions as default.